addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -153,6 +169,15 @@ function factory(dependencies) {
             this.update({ isAddingChannel: true });
         }
 
+        /**
+         * Handles click on the "Start a meeting" button.
+         *
+         * @param {MouseEvent} ev
+         */
+        onClickStartAMeetingButton(ev) {
+            this.createMeetingChannel();
+        }
```

Comment:
The message "No user found that is not already a member of this channel." is not correct in the invitation component.

After creating the channel both of these should be called:
```
channelInvitationForm.update({ doFocusOnSearchInput: true });
channelInvitationForm.searchPartnersToInvite();
```